devito/profiler.py

```diff
@@ -81,18 +86,23 @@ def add_profiling(self, code, name, byte_size=4, omp_flag=None):
                           Defaults to 4.
         :param omp_flag: OpenMP flag to add before profiling operations,
                          if needed.
+        :param to_ignore: List of strings containing the labels of
+                          symbols used as loop variables
```

Comment:
Whenever a new Iteration object is instantiated, we could update a global data structure (eg belonging to the class, not to the instances) which keeps track of all Iteration objects instantiated. Then the Profiler could just have to query Iteration (e.g. through a static method) to get all used indices

I agree it's probably pointless to extract them from cgen